src/renault_api/cli/charge/control.py

```diff
@@ -47,5 +47,31 @@ async def start(
     vehicle = await renault_vehicle.get_vehicle(
         websession=websession, ctx_data=ctx_data
     )
-    response = await vehicle.set_charge_start()
+    details = await vehicle.get_details()
+    if details.get_model_code() == "XBG1VE":
+        response = await vehicle.set_charge_pause_resume("resume")
+    else:
+        response = await vehicle.set_charge_start()
     click.echo(response.raw_data)
+
+
+@click.command()
+@click.pass_obj
+@helpers.coro_with_websession
+async def stop(
+    ctx_data: Dict[str, Any],
+    *,
+    websession: aiohttp.ClientSession,
+) -> None:
+    """Stop charge."""
+    vehicle = await renault_vehicle.get_vehicle(
+        websession=websession, ctx_data=ctx_data
+    )
+    details = await vehicle.get_details()
+    if details.get_model_code() == "XBG1VE":
+        response = await vehicle.set_charge_pause_resume("pause")
+        click.echo(response.raw_data)
+    else:
+        response = None
+        click.echo("Function unavailable")
+        exit(1)
```

Comment:
I think it is better to simply create a new function against the vehicle, and make the check there (you can raise NotImplementedException for the other vehicles) 
```suggestion
    response = await vehicle.set_charge_stop()
    click.echo(response.raw_data)
```